lib/elixir/lib/system.ex

```diff
@@ -940,8 +940,8 @@ defmodule System do
   unit before you display them to humans.
 
   To determine how many seconds the `:native` unit represents in your current
-  runtime, you can can call this function to convert 1 second to the `:native`
-  time unit: `System.convert_time_unit(1, :second, :native)`.
+  runtime, you can call this function to convert 1 second to the `:native`
+  time unit: `System.convert_time_unit(1, :second, :native)` which will return 1_000_000_000.
```

Comment:
The result of the `native` conversion is dependent on the OS, so I don't think we should include this precise example, unfortunately.

Reply:
Thanks, learned new thing. Just removed the  example.